### DISCUSSION TITLE
抽到朱鸢了

### DISCUSSION THREAD

**Discussion by MiG-2233** [share121/inter-knot#1530] [其他]
晚上好啊，马上要迎来1.1版本了，大家抽到朱鸢了吗，我这边也是运气很好，抽到朱鸢了呀，祝大家抽卡不歪，要啥出啥，共同为新艾利都建设出一份力！![Screenshot_2024-07-30-10-07-52-152_com hykb yuanshenmap](https://github.com/user-attachments/assets/1712c4ad-ea6b-49b8-b42b-516fe0a2c977)

**Comment by bin-lin2** [comment]
你怎么知道我一命格莉丝了（扭曲）

**Comment by canifish** [comment]
@ZhuYuan-148

**Comment by Mayf1ower** [comment]
你看，我就说朱鸢对狼哥的提升很大吧

**Comment by ZhuYuan-148** [comment]
![1723215847706](https://github.com/user-attachments/assets/3aaca841-1ec2-48b5-a5a6-56f12c806070)
